src/modules/clearCompleted.js

```diff
@@ -0,0 +1,14 @@
+function removeCheckedItems(Todo) {
+  // Filter out the checked items
+  const uncheckedItems = Todo.todoLists.filter((item) => !item.completed);
+
+  uncheckedItems.forEach((item, index) => {
+    item.index = index;
+  });
```

Comment:
![image (9)](https://github.com/Othniel2471/todolist-mv/assets/59707859/97300318-36c0-451a-a0ee-aecdc1a8777b)


Very well done by making changes requested by the previous reviewer but while adding the reload functionality you are updating indexes again but starting from zero while it should start from one kindly fix this issue to be ready to move to next step

Reply:
@souad988  Please I don't understand the comment in the review, please can you shed more light, because I am confused .